app/views/organisations/_promotional_features.html.erb

```diff
@@ -1,22 +1,27 @@
 <% if @documents.has_promotional_features? %>
-<section class="organisation__margin-bottom organisation__promotions">
```

Comment:
It would be good if the commit message included a bit more info as to why we are changing the class and want the `word-wrap: break-word` here and we are moving away from the previous styles.

Also since `organisation__promotions` class this is no longer used anywhere could we remove the definition? https://github.com/alphagov/collections/blob/0473b65512dc0e3f57189cce87e32ce74b90fc91/app/assets/stylesheets/views/_organisation.scss#L34-L59

Reply:
Thanks Aga, I've removed the legacy code and improved the commit message.